core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -151,6 +151,10 @@ public static InstanceType fromString(String text) {
     METRIC_REGISTRY.registerAll(new ClassLoadingGaugeSet());
     METRIC_REGISTRY.registerAll(new CachedThreadStatesGaugeSet(5, TimeUnit.SECONDS));
     METRIC_REGISTRY.registerAll(new OperationSystemGaugeSet());
+
```

Comment:
why not add it in the MemoryUsageGaugeSet?

Reply:
That part is not included